src/x-platform/electron.js

```diff
@@ -17,6 +18,24 @@ if (window.require) {
 
 const notificationClickTimeout = 20000
 
+// http://crocodillon.com/blog/parsing-emoji-unicode-in-javascript
+const emojiRegExp = new RegExp([
```

Comment:
@kof  you know, I've reconsidered that.
now replacing `:emoji:` to utf symbol is a part of markdown process in old code and if we keep 2 versions of text: one with parsed markdown and replaced emoji's and another with parsed markdown but without replaced emoji's -- that would be huge overhead for main codebase.
Instead this parsing only in case of win7 (which is I hope will die in meanwhile) is reasonable. 

so this PR is ready to merge or if you have more questions let's discuss.